app/controllers/api/v1/packages_controller.rb

```diff
@@ -161,16 +160,14 @@ def designate_stockit_item(order_id)
       end
 
       def undesignate_partial_item
-        OrdersPackage.undesignate_partially_designated_item(params[:package])
-        @package.undesignate_from_stockit_order
+        designator = Designator.new(@package, params[:package])
```

Comment:
instead of assigning it to a variable, directly use `Designator.new(@package, params[:package]).undesignate`

Reply:
Agree :+1:

Reply:
Sure will do it.